docs/sources/writing-guide/image-guidelines/index.md

```diff
@@ -225,7 +232,7 @@ The image reference that you add to the Markdown renders the image when the Graf
 </li>
 <li>Add the image reference to the Markdown file.<br><br>
 We do not expect you to test that the image renders in a local build of the help documentation. The Grafana technical documentation team will ensure that the image reference works correctly.</li><br>
-<li>Attach the image to the PR.</li>
+<li>Attach the image to the GitHub PR description.</li>
```

Comment:
@robbymilo When you get ready to merge this again (when the feature goes live) could you include some information about how the images get from the PR to the repo?  Is there some script that's running in the background on all the repos (Grafana/Loki/Mimir/Tempo) to copy/move images from that repo to the website repo?  Is there some automatically generated alert for the docs team to test the images on community PRs?

Reply:
> how the images get from the PR to the repo?

Images will no longer go to the website repo, and instead be uploaded at https://admin.grafana.com/upload/

>  Is there some script that's running in the background on all the repos (Grafana/Loki/Mimir/Tempo) to copy/move images from that repo to the website repo? 

Only the sync that copies a directory from a docs repo to the website repo. Any files, including images, are synced to the website repo.

>  Is there some automatically generated alert for the docs team to test the images on community PRs?

Once an image has been uploaded, it can be tested with the `make docs` command - see the instructions for more info.